api/graylog.go

```diff
@@ -151,7 +152,16 @@ func UpdateRegistration(httpClient *http.Client, ctx *context.Ctx, status *grayl
 		registration.NodeDetails.Status = status
 		registration.NodeDetails.Metrics = metrics
 		if len(ctx.UserConfig.ListLogFiles) > 0 {
-			registration.NodeDetails.LogFileList = common.ListFiles(ctx.UserConfig.ListLogFiles)
+			fileList := common.ListFiles(ctx.UserConfig.ListLogFiles)
+			fileListSize := (int)(unsafe.Sizeof(common.File{})) * len(fileList)
```

Comment:
Is this actually correct? As far as I understand, this measures the size of an empty `common.File` struct. But a filled struct might be much larger because we don't know the length of the strings beforehand.

How about measuring the serialized request by marshaling it into JSON?